src/devices/src/virtio/net/device.rs

```diff
@@ -543,7 +543,6 @@ impl Net {
     pub fn process_tap_rx_event(&mut self) {
         METRICS.net.rx_tap_event_count.inc();
         if self.queues[RX_INDEX].is_empty(&self.mem) {
-            error!("The RX queue is empty, there is no available buffer.");
```

Comment:
Same problem in line 575 and I expect others. Please have the fix address all instances of this behavior.